### PR TITLE
Remove italics from ComboBox and SearchAutocomplete

### PR DESCRIPTION
**Pull request by yihuiliao**
Removes the italics font when it says "Loading..." in ComboBox and "No results" in SearchAutocomplete

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx [hunk 1]
```diff
@@ -297,3 +299,48 @@ export const WithAvatars: SearchAutocompleteStory = {
     </SearchAutocomplete>
   )
 };
+
```
Comment (yihuiliao) [inline packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx:302]: added an async example of SearchAutocomplete so we could test the "No results" but if we wanna remove this lemme know

**Comment by yihuiliao** [comment]
I left the italics for StatusLight when variant="neutral" since it seems like it's in the designs but if want to remove that as well I'm happy to update this PR

**Review by reidbarber — APPROVED** [review]
LGTM, thanks for including the new story

File: packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx (hunk 1 above)
Comment (snowystinger) [reply packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx:302]: It's not showing No results for me

Comment (yihuiliao) [reply packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx:302]: Oh interesting...it shows it for me. If you type in a string like "ajdkfla" does it say it?

Comment (snowystinger) [reply packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx:302]: o, haha, I just opened it and expected it to not have any results, I can do it that way

**Review by snowystinger — APPROVED** [review]

**Pull request merged**